Samples/iOS-Swift/iOS13-Swift/AppDelegate.swift

```diff
@@ -8,8 +8,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     static func startSentry() {
         // For testing purposes, we want to be able to change the DSN and store it to disk. In a real app, you shouldn't need this behavior.
-        let dsn = DSNStorage.shared.getDSN() ?? AppDelegate.defaultDSN
-        DSNStorage.shared.saveDSN(dsn: dsn)
+        let dsn: String? = nil
```

Comment:
@brustolin  We are never setting this `dsn` variable, resulting in an empty dsn in options. I'm creating a PR to fix this.